src/sample/Models/QuestionContainer.java

```diff
@@ -0,0 +1,27 @@
+package sample.Models;
+
+import java.util.HashMap;
+
+public class QuestionContainer {
+    public HashMap<PriceCategory, Question> containerOfAllQuestions;
+    private static int numberOfAllQuestionsInContainer = 0;
+    private static final QuestionContainer instanceOfQuestionContainer = new QuestionContainer();
+
+    public void addQuestionToContainer(PriceCategory category, Question questionToAdd) {
+        containerOfAllQuestions.putIfAbsent(category, questionToAdd);
+        numberOfAllQuestionsInContainer++;
+    }
+
+    public int getNumberOfAllQuestionsInContainer() {
+        return numberOfAllQuestionsInContainer;
```

Comment:
`numberOfAllQuestionsInContainer` is not always equal to the number of items in `containerOfAllQuestions` if the code is written like this. I would suggest removing the getter for the `HashMap`, adding a getter for specific questions and removing this property.